### PR TITLE
chore:foundry test for LineaWorldID

### PR DESCRIPTION
**Pull request by Jemiiah**
issue: #4 
creating a comprehensive end-to-end test for the entire World ID state bridge system. to ensure that all components work together correctly and that the cross-chain functionality operates as expected.

### CODE REVIEW

**Comment by Jemiiah** [comment]
still working on this would complete soon

**Comment by kfastov** [comment]
Hi @Jemiiah! Could you please run `forge fmt`?

**Comment by Jemiiah** [comment]
okay ser

**Comment by Jemiiah** [comment]
i did that

File: src/LineaStateBridge.sol [hunk 1]
```diff
@@ -11,37 +11,35 @@ import { ICrossDomainOwnableLinea } from "./interfaces/ICrossDomainOwnableLinea.
...
 
-    /// @notice Amount of fee on Linea for setMessageService
-    uint256 internal _feeSetMessageService;
+    /// @notice The default gas limit amount to buy on an Linea stack chain to do simple transactions
 
-    /// @notice The default fee for Linea transactions.
-    /// Setting this to 0 means that messages will have to be claimed manually on L2
-    uint256 public constant DEFAULT_LINEA_FEE = 0;
+    /// @dev For estimation see https://github.com/kfastov/worldcoin-bridge-linea/pull/9
+    uint32 public constant DEFAULT_LINEA_GAS_LIMIT = 1_000_000;
```
Comment (kfastov) [inline src/LineaStateBridge.sol:42]: Sorry but this source file was changed in the mainstream and it seems your PR rolls back that changes. Please check the main branch and try to merge it carefully

File: .gitmodules
Comment (kfastov) [inline .gitmodules:1]: Do you have any difficulties with current dependency management? We decided to not have submodules, like it's done in the template repo. Anyway, I don't think it's in the scope of this PR. If you feel like we should migrate to submodules, please submit another PR or even an issue

File: .vscode/settings.json
Comment (kfastov) [inline .vscode/settings.json:1]: There's an error in this JSON

File: test/mocks/WorldIDIdentityManager.t.sol
Comment (kfastov) [inline test/mocks/WorldIDIdentityManager.t.sol:1]: Yeah that's cool to have this

**Review by kfastov — CHANGES_REQUESTED** [review]
I need more time to check these tests. Can you please fix the other errors at the moment?

File: src/LineaStateBridge.sol (hunk 1 above)
Comment (Jemiiah) [reply src/LineaStateBridge.sol]: I think I have resolved this

**Comment by Jemiiah** [comment]
i will close this one and create a new pr @kfastov
